src/Components/ProductTable.jsx

```diff
@@ -78,7 +78,8 @@ export default function ProductTable({ cart, updateCart }) {
                         }
                       });
                     }
-
+                    console.log(newCart);
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**nitpick:** ‏Please remove console.log